video/out/wayland_common.c

```diff
@@ -2179,8 +2179,7 @@ static void info_icc_file(void *data, struct wp_image_description_info_v1 *image
                           int32_t icc, uint32_t icc_size)
 {
     struct vo_wayland_preferred_description_info *wd = data;
-    if (wd->is_parametric)
-        return;
+    wd->is_parametric = false;
```

Comment:
Actually you can remove `is_parametric` altogether right? `info_done` can just use the presence of an icc file to determine what to do.

Reply:
Couldn't it be the case the mmap fails for whatever reason so we don't have an ICC file even though we got icc_file event? In that case we'd end up setting preferred_csp to an uninitialized struct